neighbors/src/components/about/aboutLayout.css

```diff
@@ -0,0 +1,63 @@
+.container {
```

Comment:
Mamy już klasę container w stylach globalnych, mówiłem o tym
Jak to zmerguijemy to znowu nam się rozjedzie
-albo użyj którejś metody css in js albo zmień nazwę lub użyj po prostu tego co już jest :)

Reply:
zmieniłem nazwę